static/locales/zh-TW/nav.json

```diff
@@ -3,5 +3,9 @@
   "nav.solutions": "解決方案",
   "nav.developer": "開發者",
   "nav.pricing": "價格",
-  "nav.more": "更多"
-}
\ No newline at end of file
+  "nav.more": "更多",
+  "nav.community":"社群",
+  "nav.Token":"代幣",,
```

Comment:
Could you remove the additional `,`?

Reply:
>   "nav.more": "更多",
>   "nav.community":"社群",
>   "nav.Token":"代幣",

Like this?

Reply:
yes

Reply:
You could use this online tool to validate the format. https://jsonlint.com/